app/uk/gov/hmrc/timetopay/arrangement/TTPArrangementController.scala

```diff
@@ -16,17 +16,19 @@
 
 package uk.gov.hmrc.timetopay.arrangement
 
+import javax.inject.Inject
+
 import play.api.Logger
 import play.api.libs.json.Json.toJson
-import play.api.mvc.{Result, Action, RequestHeader}
+import play.api.mvc.{Action, RequestHeader, Result}
 import uk.gov.hmrc.play.microservice.controller.BaseController
 import uk.gov.hmrc.timetopay.arrangement.services.{DesApiException, TTPArrangementService}
 
-import scala.concurrent.{Future, ExecutionContext}
+import scala.concurrent.{ExecutionContext, Future}
```

Comment:
It's not an issue, but wondering why these imports are just being moved around :P